### ISSUE TITLE
why loss starts always around 106 when trained?

### ISSUE THREAD

**Issue by qunsorber** [thtrieu/darkflow#405]
just as the question,and where to modify the epoches numbers and batch size?

**Comment by Cpruce** [comment]
Hi qunsorber,

You can supply the --epoch and --batch flags on the command line. However, I haven't found yet how many steps are in an epoch.

**Comment by Tomernudell** [comment]
@Cpruce  epoch represent a full training cycle, which can be calculated by :
Total num of training image / batch size = num steps in epoch

**Comment by andresmafladelgado** [comment]
Loss starts on different values if you use pretrained weights. Making the whole process faster.

**Comment by Cpruce** [comment]
@Tomernudell yup I forgot at the time. You're right about num steps. Those are the flags you want though

**Comment by normansiboro** [comment]
Hi @Cpruce 
Could you please explain the difference between step and epoch in detail
here is mine:
epoch : 250
batch_size = 64
total images = 100
How could I get 6 steps in every epoch?

**Comment by onurbarut** [comment]
@normansiboro 
epoch = total images / batch_size
assuming your batch_size is 2 then 1 epoch means 100 / 2 steps.
again assuming your batch_size is 2, 250 epoch takes 250 * 50 steps.

**Comment by ambr89** [comment]
@onurbarut @tommmn Really?
I think that num_step in each epoch = tot_images / subdivisions
